lib/index.py

```diff
@@ -1,5 +1,5 @@
 #!/usr/bin/env python3
-
+"""Docstring."""
```

Comment:
Same here.  We should put a better docstring if we can.

Reply:
done.